packages/components/src/ProgressBar/ProgressBar.css

```diff
@@ -1,17 +1,25 @@
 :root {
   --progress-bar--backgroundColor: var(--color-grey--lightest);
-  --progress-bar--barColor: var(--color-lime);
+  --progress-bar--barColor: var(--color-green);
+  --progress-bar--height: var(--space-base);
 }
 
-.container,
-.content {
+progress[value] {
   width: 100%;
-  height: var(--progress-bar--height);
+  /* Reset the default appearance */
+  border: none;
+  -webkit-appearance: none;
+  appearance: none;
+}
+
+progress[value]::-webkit-progress-bar {
```

Comment:
This uses the webkit prefix, there's also a -moz- prefix... do I need to add these prefixes or should autoprefixer bail me out?

Reply:
Figured it out, autoprefixer won't bail me out

- MOZ and WEBKIT actually use _different pseudo classes_ so trying to combine them into one just doesn't work

Reply:
They two pseudo classes for webkit don't have the same styling concerns as the one element for moz